pkg/repositories/interfaces/common.go

```diff
@@ -20,8 +20,9 @@ type ListResourceInput struct {
 	// MapFilters refers to primary entity filters defined as map values rather than inline sql queries.
 	// These exist to permit filtering on "IS NULL" which isn't permitted with inline filter queries and
 	// pq driver value substitution.
-	MapFilters    []common.MapFilter
-	SortParameter common.SortParameter
+	MapFilters        []common.MapFilter
+	SortParameter     common.SortParameter
+	JoinTableEntities map[common.Entity]bool
```

Comment:
can we get some documentation here, that explain what this is for?

Reply:
done